src/components/v5/frame/NavigationSidebar/NavigationSidebar.tsx

```diff
@@ -128,11 +128,14 @@ const NavigationSidebarContent: FC<NavigationSidebarProps> = ({
                   toggleOffThirdLevelMenu();
                 }}
                 className={clsx(
-                  'rounded-full border-[.1875rem] transition-all',
+                  'rounded-full border-[.1875rem] transition-all sm:hover:border-blue-400',
```

Comment:
What are those changes added here for? I didn't see it on a design

Reply:
<img width="604" alt="Screen Shot 2023-11-10 at 12 57 20" src="https://github.com/JoinColony/colonyCDapp/assets/119587095/4a539bde-5cc4-4f34-8908-41b0c7d89535">